source/Coop/Coop.csproj

```diff
@@ -241,6 +241,8 @@
     <Compile Include="Mod\DebugUtil\DebugManager.cs" />
     <Compile Include="Mod\Coop.cs" />
     <Compile Include="Mod\DebugUtil\DebugUI.cs" />
+    <Compile Include="Mod\DebugUtil\MoveTo.cs" />
```

Comment:
Why is this named MoveTo? I get the event being named that but it seems that this module is more generic than specifically movement.

Reply:
Agreed. Maybe `Replay`? 

I would even argue that's not purely a debug utility, we could use it to catch up desynced clients.